### PR TITLE
add impsort-maven-plugin

### LINKED ISSUE #539: Add impsort-maven-plugin
the formatter-maven-plugin takes care about formatting rules but import sorting is not implemented which lead to some unwanted changes in case the import order is changed by the IDE.

We could add [impsort-maven-plugin](https://code.revelc.net/impsort-maven-plugin/) to solve the issue

### PR DESCRIPTION
**Pull request by lburgazzoli**
Fixes #539

### CODE REVIEW

**Comment by metacosm** [comment]
Is there something else to do for this?

File: contributing/eclipse.importorder [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Organize Import Order
+#Mon Sep 20 14:41:51 CEST 2021
```
Comment (csviri) [inline contributing/eclipse.importorder:2]: Would remove these comments, but the order looks good!!

**Comment by csviri** [comment]
Is this plugin supported also by Intellij IDEA?

**Review by csviri — APPROVED** [review]
posted some notes, but LGTM!!
!

**Comment by lburgazzoli** [comment]
> Is this plugin supported also by Intellij IDEA? or other editors?

Yes, the Eclipse Code Formatter plugin supports this

File: contributing/eclipse.importorder (hunk 1 above)
Comment (lburgazzoli) [reply contributing/eclipse.importorder]: done

**Comment by lburgazzoli** [comment]
> Is there something else to do for this?

no, removed from "draft"

**Pull request merged**